lib/tilex/web/templates/feed/index.xml.eex

```diff
@@ -15,6 +15,9 @@
         <description>
           <![CDATA[<%= Tilex.Markdown.to_html(post.body) %>]]>
         </description>
+        <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">
```

Comment:
I agree.  My concern was a possible conflict in specifications but after further review this doesn't seem to be an issue.